includes/admin/payments/refunds.php

```diff
@@ -192,21 +202,33 @@ function edd_refund_details_attributes( $refund ) {
 	</h2>
 
 	<div class="edd-admin-box-inside edd-admin-box-inside--row">
-		<span>
-			<?php esc_html_e( 'Created', 'easy-digital-downloads' ); ?>
-		</span>
-		<time datetime="<?php echo esc_attr( EDD()->utils->date( $refund->date_created, null, true )->toDateTimeString() ); ?>">
-			<?php echo edd_date_i18n( $refund->date_created, 'M. d, Y' ) . '<br />' . edd_date_i18n( strtotime( $refund->date_created ), 'H:i' ); ?> <?php echo esc_html( edd_get_timezone_abbr() ); ?>
-		</time>
+		<div class="edd-form-group">
```

Comment:
I feel like my notifications pulled up a different version of the changes and I was seeing the order of the Created/Original Order output reversed, and then was super confused when that was not what I was seeing in the admin. My preference would be to keep the refund creation date/time at the top of the box, and then have the original order number/link underneath (partly because with the "Created" not saying "Refund Created" it would be possible to confuse that data with the original order since that's immediately above it).